### ISSUE TITLE
Is it possible to add each class test accuracy?

### ISSUE THREAD

**Issue by Chen65010445** [calmisential/TensorFlow2.0_Image_Classification#2]
Sometimes I thought only total acc would not satisfy my request, so is it possible to show each class test acc during the evaluation?

**Comment by boguss1225** [comment]
Yes see my folked repository. (https://github.com/boguss1225/TensorFlow2.0_Image_Classification)
It provides following additional features!
    1. Added 25 models from Keras API
    2. Auto Evaluation + Testing after training
    3. Added more metrics (AUC, Precision, Recall)
    4. Added early stop (patient=5) callback
    5. Added Saving Best model callback
    6. Added History saving function
